### PR TITLE
Desktop_RWD_version

### PR DESCRIPTION
**Pull request by stenlisuryadinata**
Desktop_RWD_version

required change: 
Use media queries to make sure that your website displays correctly on desktop screens.

in the following task: 
the media queries is used to develop desktop RWD version.

### CODE REVIEW

File: styles_.css [hunk 1]
```diff
@@ -0,0 +1,1069 @@
...
+  height: auto;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 12px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background: #fff;
+}
```
Comment (emmiiorji) [inline styles_.css:196]: Kindly ensure that the cards are close to the design as possible. Consider the comparison below:

Your design:
image

Figma design:
image

File: styles_.css [hunk 2]
```diff
@@ -0,0 +1,1069 @@
...
+    transform: rotate(90deg);
+    position: absolute;
+    right: 0;
+  }
+
+  .mylanguage-keyfeatures-tech {
+    display: flex;
+    flex-direction: row;
+  }
+
```
Comment (emmiiorji) [inline styles_.css:886]: Kindly adjust the about myself section to match the design. The languages have icons within the boxes and the horizontal lines below skills and frameworks titles are faded.

Your design:
image

Figma design:
image

File: styles_.css [hunk 3]
```diff
@@ -0,0 +1,1069 @@
...
+  .navbar {
+    align-self: center;
+    margin-right: 6%;
+  }
+
+  .mypage {
+    height: 90vh;
+  }
+
+  .mypage-container {
```
Comment (emmiiorji) [inline styles_.css:970]: Kindly ensure that the mobile version is responsive until the desktop screen size.

image

image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @stenlisuryadinata, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- Proper use of media queries to display desktop version
- You have the project image and description alternating as in design


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by stenlisuryadinata** [comment]
required changes : 
-that the cards are close to the design as possible. Consider the comparison between figma and the design
-the languages have icons within the boxes and the horizontal lines below skills and frameworks titles are faded
-the mobile version is responsive until the desktop screen size.

in the following task 
-change the cards as per figma 
-the icons are added and the horizontal line are faded as per figma
-the mobile person is responsive until the desktop screen size

**Review by bellom — APPROVED** [review]
@stenlisuryadinata,

### This is a manual review

# Approved :+1:

Cheers and Happy coding!:clap::clap::clap:

**Comment by stenlisuryadinata** [comment]
Thank you, I really appreciated it

**Pull request merged**